### ISSUE TITLE
JSON output gives incorrect error lines

### ISSUE THREAD

**Issue by MVrachev** [palantir/tslint#4604]
### Bug Report

I found a bug with the starting line reported by the JSON output provided by TSLint.

- __TSLint version__: 5.14.0
- __TypeScript version__: 3.3.4000
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

Line 55 is linted from a security plugin of TSLint: https://github.com/webschik/tslint-config-security

![image](https://user-images.githubusercontent.com/16246778/54990155-b9be0a80-4fc2-11e9-9b49-d1e3cfcbc1ba.png)

with `tslint.json` configuration:

```json
{
    "extends": ["tslint-config-security"],
    "jsRules": {
      "tsr-detect-buffer-noassert": [true],
      "tsr-detect-child-process": [true],
      "tsr-detect-eval-with-expression": [true],
      "tsr-detect-no-csrf-before-method-override": [true],
      "tsr-detect-non-literal-buffer": [true],
      "tsr-detect-non-literal-fs-filename": [true],
      "tsr-detect-non-literal-regexp": [true],
      "tsr-detect-non-literal-require": [true],
      "tsr-detect-possible-timing-attacks": [true],
      "tsr-detect-pseudo-random-bytes": [true],
      "tsr-detect-unsafe-regexp": [true],
      "tsr-disable-mustache-escape": [true],
      "tsr-detect-html-injection": [true],
      "tsr-detect-sql-literal-injection": [true],
      "tsr-detect-unsafe-cross-origin-communication": [true],
      "tsr-detect-unsafe-properties-access": [true]
  }
}
```

#### Actual behavior

When I run tslint with upon a file like this:
``` 
tslint -c tslint/tslint.json cache.js
```

I get this result:
``` 
ERROR: cache.js:55:3 - Found fs.writeFileSync with non-literal argument at index 0 
```

When I run tslint like this 

```
tslint -c tslint/tslint.json cache.js -t json
```

I get this JSON output: 
```
[  
   {  
      "endPosition":{  
         "character":18,
         "line":54,
         "position":1920
      },
      "failure":"Found fs.writeFileSync with non-literal argument at index 0",
      "name":"cache.js",
      "ruleName":"tsr-detect-non-literal-fs-filename",
      "ruleSeverity":"ERROR",
      "startPosition":{  
         "character":2,
         "line":54,
         "position":1904
      }
   }
]
```

As you can clearly see "startline" in the JSON output is different from the line provided in the default output.

#### Expected behavior
No difference between startline in the JSON output and the reported line in the default output.

**Comment by JoshuaKGoldberg** [comment]
Duplicate of #4528.
